testnets/_IBC/synternettestnet-osmosistestnet.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "../ibc_data.schema.json",
```

Comment:
this relative path requires and an extra ../ (to go up two directories)
`"$schema": "../../ibc_data.schema.json",`

Reply:
Thanks! Fixed.